en/faq.md

```diff
@@ -6,110 +6,110 @@ sidebarDepth: 2
 
-如果你确定是误封，请加入 [官方用户交流群](/user-group.html) 或 [发送邮件工单](/email.html)，联系站点管理员处理。
+If you are sure that it was blocked by mistake, please join the [Official User Exchange Group](/user-group.html) or [Send an Email Ticket](/email.html), and contact the site administrator for handling.
 
-## 客户端加载相关
+## Client loading related
 
-### 为什么我在网站上设置好了材质，但是在游戏中不显示 / 没更新？
+### Why do I set up the material on the website, but it doesn't show/update in game?
 
-这是个很常见的问题。大多数情况下，这个问题是以下四个原因引起的：
+This is a very common problem. In most cases, this problem is caused by the following four reasons:
 
-1. 你没有正确地安装和配置皮肤 Mod；
-2. 你安装的其它 Mod（如 NonUpdate）干扰了皮肤 Mod 与 LittleSkin 之间的连接；
-3. 你的材质被你的皮肤 Mod 缓存了；
-4. 你使用了 Alex 模型的皮肤，并且你的 Minecraft 版本低于 1.8。
+1. You did not install and configure the skin mod correctly;
+2. Other mods you installed (such as NonUpdate) interfere with the connection between the skin mod and LittleSkin;
+3. Your material is cached by your skin mod;
+4. You are using the Alex model skin and your Minecraft version is lower than 1.8.
 
-解决方案分别是：
+The solutions are:
 
-1. 正确安装并配置皮肤 Mod。你可以在 [新手指引 - 配置 Mod](/newbee/mod.html) 中学习如何正确配置皮肤 Mod；
-2. 删除这些 Mod，或让它们绕过皮肤 Mod 与 LittleSkin 之间的连接；
-3. 等待几分钟后再试。如果还是没有更新，尝试清除你的皮肤 Mod 的缓存（即删除对应的文件夹）：
-    - CustomSkinLoader 的缓存位于 `.minecraft/CustomSkinLoader/caches`；
-    - SkinPort 的缓存位于 `.minecraft/cachedImages`
-4. 更换皮肤 Mod，或更换为 Steve 模型的材质：
-    - 对于 1.7.10，请使用 SkinPort 加载材质；
-    - 对于更低版本，目前无解，请更换为 Steve 模型的皮肤。
+1. Install and configure the skin mod correctly. You can learn how to properly configure the skin mod in [Beginner's Guide - Configuring Mod](/newbee/mod.html);
+2. Remove these mods, or let them bypass the connection between the skin mod and LittleSkin;
+3. Wait a few minutes and try again. If it still doesn't update, try clearing your skin mod's cache (ie deleting the corresponding folder):
+    - CustomSkinLoader's cache is located in `.minecraft/CustomSkinLoader/caches`;
+    - SkinPort's cache is located in `.minecraft/cachedImages`
+4. Change the skin mod, or change to the material of the Steve model:
+    - For 1.7.10, use SkinPort to load materials;
+    - For earlier versions, there is currently no solution, please change to the skin of the Steve model.
 
-如果你确定你的问题不是以上原因引起的，或者你按照以上的解决方案做了之后你的角色的材质依然没有显示或更新，请在详细阅读 [报告问题的正确姿势](/report.html) 后带上你的皮肤 Mod 的日志，加入 [官方用户交流群](/user-group.html) 或 [发送邮件工单](/email.html) 询问，或直接购买一对一远程技术支持服务。
+If you are sure that your problem is not caused by the above reasons, or after you follow the above solutions, the texture of your character still does not display or update, please read [Report the correct pose for problems](/report.html) Then bring your skin mod log, join the [official user exchange group](/user-group.html) or [send an email ticket](/email.html) to inquire, or directly purchase one-to-one remote technical support services .
```

Comment:
**AND AGAIN**, check the title/subtitle of another file before using it as a reference.

```suggestion
If you are sure that your problem is not caused by the above reasons, or after you follow the above solutions, the texture of your character still does not display or update, please read [How to report a problem](/report.html) Then bring your skin mod log, join the [official user group](/user-group.html) or [send an email ticket](/email.html) to get help, or directly purchase an one-to-one remote technical support services.
```